build.gradle

```diff
@@ -33,10 +35,15 @@ project(":DungeoneerDesktop") {
 
     dependencies {
         implementation project(":Dungeoneer")
-        api "com.badlogicgames.gdx:gdx-backend-lwjgl:$gdxVersion"
+        api "com.badlogicgames.gdx:gdx-backend-lwjgl3:$gdxVersion"
+        api "com.badlogicgames.gdx:gdx-lwjgl3-angle:$gdxVersion"
+        // Can't use this next yet due to: https://github.com/libgdx/libgdx/issues/6989
+        //api "com.badlogicgames.gdx:gdx-lwjgl3-glfw-awt-macos:$gdxVersion" // Needed for Mac to start on the correct thread
         api "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"
-        api "com.badlogicgames.gdx:gdx-controllers-desktop:$gdxVersion"
-        api "com.badlogicgames.gdx:gdx-controllers-platform:$gdxVersion:natives-desktop"
+        api "com.badlogicgames.gdx-controllers:gdx-controllers-desktop:$gdxControllersVersion"
+        implementation("com.badlogicgames.jamepad:jamepad:$jamepadVersion") {
+            exclude group: 'com.badlogicgames.gdx', module: 'gdx-jnigen-loader'
+        }
```

Comment:
I think we can remove this? @Interrupt confirmed that it wasn't needed I believe?